.github/workflows/ci.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - name: Run cargo check (without dev-dependencies to catch missing feature flags)
         if: startsWith(matrix.rust, 'nightly')
         run: cargo check -Z features=dev_dep
-      - run: cargo test
+      - run: cargo test --all-features
```

Comment:
We should also test with no features enabled.

Reply:
Done.